spec/datetimeformat.html

```diff
@@ -491,21 +491,21 @@ <h1>Internal slots</h1>
       </emu-note>
 
       <p>
-        The value of the [[LocaleData]] internal slot is implementation defined within the constraints described in <emu-xref href="#sec-internal-slots"></emu-xref> and the following additional constraints:
+        The value of the [[LocaleData]] internal slot is implementation defined within the constraints described in <emu-xref href="#sec-internal-slots"></emu-xref> and the following additional constraints, for all locale values _locale_:
       </p>
 
       <ul>
         <li>
-          The list that is the value of the `"nu"` field of any locale field of [[LocaleData]] must not include the values `"native"`, `"traditio"`, or `"finance"`.
+          [[LocaleData]].[[&lt;_locale_&gt;]].[[nu]] must be a list that does not include the values `"native"`, `"traditio"`, or `"finance"`.
```

Comment:
```suggestion
          [[LocaleData]].[[&lt;_locale_&gt;]].[[nu]] must be a List that does not include the values `"native"`, `"traditio"`, or `"finance"`.
```
should this really be “traditio” or is it meant to be “traditional” or similar? Right now it looks like a typo/truncation.